CONTRIBUTING.md

```diff
@@ -0,0 +1,20 @@
+Issues
+------
+
+If you've found a bug or have a great idea for a new feature that will help **all users** of impress.js, let us know by [adding your suggestion]
+(http://github.com/bartaz/impress.js/issues/new) to the [issue tracker](https://github.com/bartaz/impress.js/issues).
+
+Guidelines:
+
+* If reporting a bug, please provide a [simplified example](http://sscce.org/) on [Pastebin](http://pastebin.com/) or [JsFiddle](https://jsfiddle.net/).
+
+Pull Requests
+-------------
+
+If you have fixed a bug or implemented a feature that you'd like to share, send your [Pull Request](https://help.github.com/articles/using-pull-requests/) against the [master branch]
+(http://github.com/bartaz/impress.js/tree/master). But remember that the team will only accept code that fits the purpose of impress.js
```

Comment:
![https-all-the-things](https://cloud.githubusercontent.com/assets/835857/13252269/e6d12bb2-da89-11e5-8563-590c45fddf30.jpg)